pom.xml

```diff
@@ -59,6 +52,10 @@
           <id>ome.maven</id>
           <url>http://artifacts.openmicroscopy.org/artifactory/maven/</url>
         </repository>
+        <repository>
```

Comment:
But this line is mandatory for testing the PR via Travis. Once the pom is pushed to release, happy to revert it.